### PR TITLE
identity optimization

### PR DESCRIPTION
**Pull request by KanishkaRajputd**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -227,35 +223,11 @@ const Home: FC<IHomeProps> = ({ latestPosts, network, networkSocialsData }) => {
...
-				}
-			})
-			.then((unsub) => {
-				unsubscribe = unsub;
-			})
-			.catch((e) => console.error(e));
-
-		return () => unsubscribe && unsubscribe();
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [network, api, apiReady, userId]);
```
Comment (alphainfinitus) [inline pages/index.tsx:232]: we're setting network every time api or user changes.
This does not make sense.

File: pages/index.tsx [hunk 2]
```diff
@@ -290,12 +262,6 @@ const Home: FC<IHomeProps> = ({ latestPosts, network, networkSocialsData }) => {
 			<main>
 				<div className='mr-2 flex justify-between'>
 					<h1 className='mx-2 text-2xl font-semibold leading-9 text-bodyBlue dark:text-blue-dark-high'>Overview</h1>
-					{isIdentityUnverified && onchainIdentitySupportedNetwork.includes(network) && (
```
Comment (alphainfinitus) [inline pages/index.tsx:293]: Are we removing the warning ?

File: src/components/OnchainIdentity/SocialVerificationInprogress.tsx
Comment (alphainfinitus) [inline src/components/OnchainIdentity/SocialVerificationInprogress.tsx:1]: Are these localstorage key changes compatible with what users might already have on their localstorage ?

**Review by alphainfinitus — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 2 above)
Comment (KanishkaRajputd) [reply pages/index.tsx:293]: this is not mandatory

File: src/components/OnchainIdentity/SocialVerificationInprogress.tsx
Comment (KanishkaRajputd) [reply src/components/OnchainIdentity/SocialVerificationInprogress.tsx:1]: yes

**Pull request merged**